terraform/context.go

```diff
@@ -138,7 +138,18 @@ func NewContext(opts *ContextOpts) (*Context, tfdiags.Diagnostics) {
 	// Determine parallelism, default to 10. We do this both to limit
 	// CPU pressure but also to have an extra guard against rate throttling
 	// from providers.
+	// We throw an error in case of negative parallelism or
+	// parallelism equal to 0.
```

Comment:
Comment is out of sync, but otherwise this is 👍

Reply:
Fixed in a4831b4 :smile: